<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="lineEdit_nickname">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>昵称</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_passwd">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>密码</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_nota">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>确认密码</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_register">
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_userid">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
